common_cartridge_parser.gemspec

```diff
@@ -5,11 +5,11 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'common_cartridge/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = 'common_cartridge'
+  spec.name          = 'common_cartridge_parser'
   spec.version       = CommonCartridge::VERSION
   spec.authors       = ['Josh Simpson', 'Remy Obein']
   spec.email         = ['jsimpson@instructure.com', 'remy@cassia.tech']
-  spec.summary       = %q{CommonCartridge}
+  spec.summary       = %q{CommonCartridgeParser}
   spec.description   = 'Parse IMS Common Cartridge packages'
   spec.homepage      = 'http://www.instructure.com/'
```

Comment:
can you please set the home page to https://github.com/vhl/common_cartridge_parser/